app/controllers/application_controller.rb

```diff
@@ -56,9 +56,8 @@ def current_team
   helper_method :current_team
 
   def after_sign_in_path_for(resource)
-    if resource.authy_enabled && resource.authy_id.blank?
-      return send("#{resource_name}_enable_authy_path")
-    end
+    return super if resource.is_a?(AdminUser)
+    return send("#{resource_name}_enable_authy_path") if resource.authy_enabled && resource.authy_id.blank?
```

Comment:
We could also move this method to its on class, so we can't better unit test this behavior 💅

Reply:
Also, this change will help to solve the codeclimate issue aswell...